src/main/java/com/glencoesoftware/bioformats2raw/Converter.java

```diff
@@ -2681,7 +2681,13 @@ private Quantity getScale(
       case 'z':
         return meta.getPixelsPhysicalSizeZ(seriesIndex);
       case 't':
-        return meta.getPixelsTimeIncrement(seriesIndex);
+        Quantity timeIncrement = meta.getPixelsTimeIncrement(seriesIndex);
+        if (timeIncrement != null && timeIncrement.value().doubleValue() > 0) {
+          return meta.getPixelsTimeIncrement(seriesIndex);
```

Comment:
Any reason not to just `return timeIncrement` here instead of calling the getter again?